src/i_video.c

```diff
@@ -1090,7 +1105,7 @@ static void ResetResolution(int height)
     // 1278x720.
 
     double vertscale = (double)actualheight / (double)unscaled_actualheight;
-    video.width = (int)(video.unscaledw * vertscale);
+    video.width = ceil(video.unscaledw * vertscale);
```

Comment:
Could you explicitly cast the `ceil()` result? The warning that forbids this has been removed.

Reply:
Done in https://github.com/fabiangreffrath/woof/pull/1402/commits/2b29191f493e413e4e9ad2577cc9f10cbc72577c